File: src/java/net/intensicode/tools/bfm/ui/RenderPanel.java
package net.intensicode.tools.bfm.ui;

import net.intensicode.tools.bfm.UiController;

import javax.swing.*;
import javax.swing.border.TitledBorder;
import java.awt.*;

public final class RenderPanel extends JPanel
    {
    public RenderPanel( final UiController aController )
        {
        setLayout( new BorderLayout() );
        setBorder( new TitledBorder( "Render Output" ) );
        final FontCellRenderPanel renderPanel = new FontCellRenderPanel( aController );
        final JScrollPane scrollPane = new JScrollPane( renderPanel );
        add( scrollPane, BorderLayout.CENTER );
        }
    }

File: src/jruby/bfm/ui/render_panel.rb
require 'bfm/ui/font_cell_render_panel'

module BFM

  module UI

    import java.awt.BorderLayout
    import javax.swing.JPanel
    import javax.swing.JScrollPane
    import javax.swing.border.TitledBorder

    class RenderPanel < JPanel

      def initialize(controller, renderer)
        super()
        set_layout BorderLayout.new
        set_border TitledBorder.new("Render Output")
        render_panel = FontCellRenderPanel.new(controller, renderer)
        scroll_pane = JScrollPane.new(render_panel)
        add scroll_pane, BorderLayout::CENTER
      end

    end

  end

end
